### PR TITLE
AST-438 - imprv: removed static CSS related to comments and load from dynamic

### PR DESCRIPTION
**Pull request by sagarb90**
### Description
- Removed static CSS related to comments and load from dynamic.
- Also removed duplicate CSS from dynamic CSS.
- https://wp-astra.atlassian.net/browse/AST-438

### Types of changes
- Improvements

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: inc/dynamic-css/comments-flex.php [hunk 1]
```diff
@@ -337,8 +344,21 @@ function astra_comments_css( $dynamic_css ) {
...
 
+		if ( $is_site_rtl ) {
+			$global_button_comment_mobile['.ast-comment-list .children'] = array(
+				'margin-right' => '0.66666em',
+			);
+		} else {
+			$global_button_comment_mobile['.ast-comment-list .children'] = array(
+				'margin-left' => '0.66666em',
+			);
+		}
```
Comment (premanshup) [inline inc/dynamic-css/comments-flex.php:360]: @sagarb90 Why are we adding this additional CSS for comment?

File: inc/dynamic-css/comments-flex.php [hunk 2]
```diff
@@ -371,6 +391,16 @@ function astra_comments_css( $dynamic_css ) {
...
+
+		if ( $is_site_rtl ) {
+			$global_button_comment_tablet['.ast-comment-avatar-wrap'] = array(
+				'margin-left' => '0.5em',
+			);
+		} else {
+			$global_button_comment_tablet['.ast-comment-avatar-wrap'] = array(
+				'margin-right' => '0.5em',
+			);
+		}
```
Comment (premanshup) [inline inc/dynamic-css/comments-flex.php:403]: @sagarb90 Here as well.

File: inc/dynamic-css/comments.php [hunk 3]
```diff
@@ -239,15 +240,23 @@ function astra_comments_css( $dynamic_css ) {
       }
 
       @media (max-width: 1200px) {
-      .ast-separate-container .comment-respond {
-        padding: 3em 2.34em;
-      }
+        .ast-separate-container .comment-respond {
+          padding: 3em 2.34em;
+        }
```
Comment (premanshup) [inline inc/dynamic-css/comments.php:245]:
```suggestion
      .ast-separate-container .comment-respond {
        padding: 3em 2.34em;
      }
```

**Review by premanshup — CHANGES_REQUESTED** [review]

File: inc/dynamic-css/comments-flex.php (hunk 1 above)
Comment (sagarb90) [reply inc/dynamic-css/comments-flex.php:360]: This is not additional CSS this was in the main `astra_dynamic_css` file and have moved it to the comment CSS file.

File: inc/dynamic-css/comments-flex.php (hunk 2 above)
Comment (sagarb90) [reply inc/dynamic-css/comments-flex.php:403]: This is not additional CSS this was in the main `astra_dynamic_css` file and have moved it to the comment CSS file.

Comment (premanshup) [reply inc/dynamic-css/comments-flex.php:403]: Okay 👍🏼

**Pull request merged**